Sources/BezierSwift/MasterComponent/BezierDialog/BezierDialog.swift

```diff
@@ -6,15 +6,226 @@
 //
 
 import SwiftUI
+import Dispatch
 
-struct BezierDialog: View {
-    var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
-    }
+private enum Metric {
+  static let dimSideMinPadding = CGFloat(40)
+
+  static let dialogMaxWidth = CGFloat(480)
+  static let dialogPadding = CGFloat(16)
+
+  static let upperStackContainerTop = CGFloat(4)
+  static let upperStackSpace = CGFloat(8)
+
+  static let middleStackTop = CGFloat(16)
+  static let middleStackSpace = CGFloat(12)
+
+  static let belowStackTop = CGFloat(20)
+  static let belowStackSpace = CGFloat(8)
+}
+
+public enum BezierDialogButtonInfo {
+  case vertical([BezierButton])
+  case horizontal([BezierButton])
+}
+
+public struct BezierDialogParam {
+  var id = UUID()
+  
+  // TODO: priority 일정 enum으로 제한두기. by jam 2023.02.24
+  var priority: Int
+  var title: String
+  var description: String
+  var buttonInfo: BezierDialogButtonInfo?
+  var dismissCompletion: (() -> Void)?
+  
+  init(priority: Int, title: String?, description: String?, buttonInfo: BezierDialogButtonInfo?, dismissCompletion: (() -> Void)?) {
```

Comment:
```suggestion
  public init(priority: Int, title: String?, description: String?, buttonInfo: BezierDialogButtonInfo?, dismissCompletion: (() -> Void)?) {
```
param init이 public 으로 되어있어야할것 같습니다.